### PR TITLE
Navbrand

### LINKED ISSUE #58: Brand-logo on the navbar
**Is your feature request related to a problem? Please describe.**
Navbar contain brand name as a simple text. Having a brand image instead of simple text would look better.

**Describe the solution you'd like**
Adding the logo as the brand-image.

**Are you willing to contribute to this issue?** [Yes/No]
Yes, I want to contribute to this issue as a part of GSSOC'21.

**Additional context**
- Before change: 
Screenshot (240)

- After change: 
Screenshot (241)

### PR DESCRIPTION
**Pull request by SandeepKrSuman**
## Description
- Changes : (i) Added an image tag under nav tag inside `./templates/mainapp/layout.html` (ii) Added a .png image inside `./static/mainapp/png` which is same as the logo image but with the white background removed.
- closes #58 

## Affected Dependencies
No dependencies have been affected. Only a png image is added inside `./static/mainapp/png`

## How has this been tested?
Tested locally for responsiveness.

## GIF Before
before

## GIF After
updated__gif

SS for small device responsiveness: 
Screenshot (254)

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

### CODE REVIEW

File: templates/mainapp/layout.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
     <header>
         <div id="topheader">
             <nav class="navbar navbar-expand-md navbar-light">
-                <a class="navbar-brand head-text" href="/"><img width="150px" src="https://user-images.githubusercontent.com/45713796/98271308-d18aac80-1fb5-11eb-9db3-dda942cc1b07.png"></a>
+                <a class="navbar-brand head-text" href="/"><img width="200px" src="./static/mainapp/png/brand-img.png"></a>
```
Comment (maneprajakta) [inline templates/mainapp/layout.html:42]: Change the src to [this way](https://github.com/Praful932/Kitabe/blob/91c168d1bcd5888c835fc48bf3ff3e0ac05064b7/templates/mainapp/index.html#L19)  it is the way to specify the path of the static way in Django template.

Comment (maneprajakta) [inline templates/mainapp/layout.html:42]: The width seems more to me from screenshots reduce it a bit.

**Review by maneprajakta — COMMENTED** [review]
@SandeepKrSuman rest all seems great :rocket:

**Review by maneprajakta — APPROVED** [review]
Good to go :rocket:

**Review by Praful932 — CHANGES_REQUESTED** [review]
@SandeepKrSuman Could you update the GIFs with the latest GIFs also in responsive modes

**Comment by SandeepKrSuman** [comment]
> @SandeepKrSuman Could you update the GIFs with the latest GIFs also in responsive modes


ezgif com-gif-maker

**Comment by maneprajakta** [comment]
> > @SandeepKrSuman Could you update the GIFs with the latest GIFs also in responsive modes
> 
> ezgif com-gif-maker

@SandeepKrSuman we have deployed your PR which you can visit [here](https://kitabe-pipeline-pr-61.herokuapp.com/)  it doesn't seem responsive for the small screen. Just check once from your end am I wrong. 
ss :
Screenshot from 2021-03-12 09-54-41


Also, you are supposed to update the ss in the PR template [this one](https://github.com/Praful932/Kitabe/pull/61#issue-590063183) edit it.

**Comment by SandeepKrSuman** [comment]
@maneprajakta I have made some changes for responsiveness in smaller devices. Added media query or the image width. 

Now it should be fine. 

Take a look: 
Screenshot (243)

File: static/mainapp/css/layout.css [hunk 2]
```diff
@@ -16,6 +16,9 @@ body {
   flex-direction: column !important;
   align-items: center !important;
 }
+.navbrandimg{
+  width: 100px;
```
Comment (maneprajakta) [inline static/mainapp/css/layout.css:20]: The width of approx 130px suits better for this, the logo is appearing too small in large screen mode.

File: static/mainapp/css/layout.css [hunk 3]
```diff
@@ -164,6 +167,10 @@ footer a:hover {
   .navbar-brand {
     width: 8px;
   }
+  .navbrandimg{
+    width: 50px;
```
Comment (maneprajakta) [inline static/mainapp/css/layout.css:171]: Choose a width size appearing too small in small screen mode.

File: templates/mainapp/layout.html [hunk 4]
```diff
@@ -39,7 +39,7 @@
     <header>
         <div id="topheader">
             <nav class="navbar navbar-expand-md navbar-light">
-                <a class="navbar-brand head-text" href="/"><img width="100px" src="{% static 'mainapp/png/brand-img.png' %}"></a>
+                <a class="navbar-brand head-text" href="/"><img class="navbrandimg" src="{% static 'mainapp/png/brand-img.png' %}"></a>
```
Comment (maneprajakta) [inline templates/mainapp/layout.html:42]: As the head-text class is not needed now remove it and also associated CSS with it.

**Review by maneprajakta — CHANGES_REQUESTED** [review]
image width issue not of appropriate size.

**Comment by SandeepKrSuman** [comment]
Please view the deployment on physical smaller-device(phone) too. Now it seems completely fine to me.

**Review by maneprajakta — APPROVED** [review]
Everything seems great to me.

**Review by Praful932 — CHANGES_REQUESTED** [review]
Great job so far @SandeepKrSuman.
[Your changes](https://kitabe-pipeline-pr-61.herokuapp.com/). [Master branch](https://kitabe-app.herokuapp.com/).
If you could see the the navbar height in both of them, the one in yours is a bit wider, if you could make the changes to reduce it to be of same size by making the kitabe icon a bit smaller so both of the navbars have the same even after icon is changed on all devices.
After that I would be happy to merge :) 

Here's an image for your reference
image

**Comment by SandeepKrSuman** [comment]
hey @Praful932 , I have adjusted the size of kitabe icon and now the height of navbar seems similar in both master and this branch. Please have a look.

**Review by Praful932 — APPROVED** [review]

**Comment by Praful932** [comment]
Fantastic work LGTM @SandeepKrSuman 🚀
I'll merge this as soon as it is approved!

**Review by maneprajakta — APPROVED** [review]
Perfect image width.
Good to go :rocket: .

**Comment by SandeepKrSuman** [comment]
Thankyou so much. 
Great that you guys were constantly guiding. Hope that this little contribution will add something to the project. Thanks a lot 😊

**Pull request merged**